src/main/java/com/github/pires/obd/commands/protocol/SpacesOffCommand.java

```diff
@@ -0,0 +1,32 @@
+package com.github.pires.obd.commands.protocol;
+
+import android.os.Parcel;
```

Comment:
This is a pure Java library. I can' accept Android code imports. Also, it seems it's not even needed.